src/periph/motion_controller/bsl_motion_controller.cpp

```diff
@@ -323,6 +328,18 @@ void BSLMotionController::handleGcode(const QString &gcode) {
                 is_handling_high_speed_ = true;
             }
             return;
+        } else if (type == "WS") {
+            settings.wobble_step = value.toDouble();
+        } else if (type == "WD") {
+            settings.wobble_diameter = value.toDouble();
+            if (settings.wobble_step > 0 && settings.wobble_diameter > 0) {
+                lcs_set_wobble_mode(settings.wobble_diameter,
+                                    settings.wobble_diameter,
+                                    settings.wobble_step,
+                                    WobbleType::WT_WHEEL);
+            } else {
+                lcs_set_wobble_mode(0, 0, 0, WobbleType::WT_DISABLE);
+            }
```

Comment:
this only accept gcode in the order `WS` first then `WD` , would it be better to handle this after the `if` and check if `WS` or `WD` is changed?